check_haproxy.rb

```diff
@@ -85,16 +91,28 @@
   exit UNKNOWN
 end
 
+
+tries = 2
+
 begin
   f = open(options.url, :http_basic_authentication => [options.user, options.password])
 rescue OpenURI::HTTPError => e
   puts "ERROR: #{e.message}"
-  exit UNKNOWN
+  exit CRITICAL
```

Comment:
These are unknown because it can't connect to haproxy, not because haproxy has too many open sessions.  Possibly temporary connection issues should not cause a CRITICAL result.